app/models/ad.rb

```diff
@@ -28,32 +39,14 @@ class Ad < ApplicationRecord
   validates :consent, presence: { message: "mora biti odobrena" }
   validates :address, presence: { message: "ne smije biti prazna" }, on: %i[create update]
   validates :email, format: /@/, if: -> { email.present? }
-  validates :service, presence: true, inclusion: { in: SERVICES }
+  validates :categories, presence: true, inclusion: { in: CATEGORIES }
```

Comment:
`validates :category`

Reply:
Great catch, fixed in https://github.com/vlado/earthquake-croatia/pull/96/commits/c756897e10aa518db6ebc3d3a4986ac30e2aceea